### ISSUE TITLE
Check for stray locking keys at the end of locking tests

### ISSUE THREAD

**Issue by PVince81** [owncloud/smashbox#122]
Not sure how, but manually one can run `redis-cli` and then `keys *` to check for stray locking keys.

I suggest that the tests regarding locking have an additional check at the end to make sure that there are no remaining keys in redis, because such keys could cause issues.

@jnfrmarks @nickvergessen

**Comment by jnfrmarks** [comment]
@jvillafanez 

JP, can you add this to your tests?

**Comment by nickvergessen** [comment]
@jnfrmarks no, this needs to be part of all tests, just like with the scrapeLog stuff

**Comment by jvillafanez** [comment]
We should replicate how the redis keys for locking are generated. I don't think it's a good idea to check `redis-cli keys '*'` because it's possible there might be other keys not related with locking. I'd prefer to narrow down the keys as much as possible.

I'm not fan of using the command-line if it's possible to use a library (https://pypi.python.org/pypi/redis for example). However, taking into account that I don't expect any additional action against redis, I'd use the command-line.

**Comment by PVince81** [comment]
In case it helps, I used this snippet recently:

``` python
import redis
red = redis.Redis(host='localhost', db=0)
keys = red.keys()
vals = red.mget(keys)
kv = zip(keys, vals)
for key, value in kv:
    print(key, ': ', value)
```

The locking keys have the word "locking" in them.

**Comment by nickvergessen** [comment]
Note: we don't use redis on our jenkins instance, so ... needs to work for all of them I guess?
